src/renderer/components/input/input.tsx

```diff
@@ -283,6 +283,9 @@ export class Input extends React.Component<InputProps, State> {
       } else {
         this.setDirty();
       }
+
+      //pressing enter indicates that the edit is complete, we can unfocus now
+      this.blur();
```

Comment:
Shouldn't this happen a few lines up, after the `this.setState()` call?

Reply:
It depends. doing it like this, the user can unfocus the input even if it's not valid. If we blurred the input right after calling setState, the user would lose the ability to do so (even if it doesn't make sense to do so).. Idk which solution is better.

Reply:
How does this work with components like `EditableList`? Have you tried that?

makes sense

Reply:
i've tested EditableLists using the accessible namespaces. The namespace is added and after that, the input blurs. Should we keep the input focussed in those cases?

Reply:
I think we should

Reply:
What's the cleanest way of doing this in this scenario? Checking the type of Input in the onKeyDown method seems pretty unclean, but that would mean each class would have to decide if the input needs to be blurred itself, or needs to  focus the input again. All of these solutions are not optimal imho

Reply:
Since I think `EditableList` is the only place where this matters I would make a prop called something like `blurOnEnter` with default value `true` and then `EditableList` can set it to `false`.